cmd/crank/beta/trace/internal/resource/xpkg/client.go

```diff
@@ -316,7 +317,8 @@ func (kc *Client) getPackageDeps(ctx context.Context, node *resource.Resource, l
 			}
 		}
 
-		dep, err := kc.getDependencyRef(ctx, lock, d.Type, d.Package)
+		// TODO(negz): Handle apiVersion and kind
```

Comment:
Not sure what exactly this TODO is about but in my experiments I could define a dependency to a custom/external package type from a Configuration and it worked fine 🤔

Reply:
Did `crank beta trace` work fine?

I don't really use that command and I'm not familiar with its code. It does appear to rely on dependency types to look up package dependencies in the lock file though.

Reply:
Pushed a commit that to support GVK style dependencies in `crank beta trace`.